### ISSUE TITLE
Build error -pg and -fomit-frame-pointer are incompatible

### ISSUE THREAD

**Issue by jonaschl** [esnet/iperf#337]
Hello,
i try to build iperf 3.1.1 and i get the following error
```
ipfire build chroot (i586) root:/iperf-3.1.1# make
Making all in src
make[1]: Entering directory '/iperf-3.1.1/src'
make  all-am
make[2]: Entering directory '/iperf-3.1.1/src'
gcc -DHAVE_CONFIG_H -I.    -pg -g -O2 -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fPIC -fstack-protector-all --param=ssp-buffer-size=4 -march=i586 -mtune=generic -fomit-frame-pointer -Wall -MT iperf3_profile-main.o -MD -MP -MF .deps/iperf3_profile-main.Tpo -c -o iperf3_profile-main.o `test -f 'main.c' || echo './'`main.c
gcc: -pg and -fomit-frame-pointer are incompatible
Makefile:877: recipe for target 'iperf3_profile-main.o' failed
make[2]: *** [iperf3_profile-main.o] Error 1
make[2]: Leaving directory '/iperf-3.1.1/src'
Makefile:632: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/iperf-3.1.1/src'
Makefile:381: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```
Remove the -pg flag in src/Makefile  src/Makefile.in  src/Makefile.am fix this error, but is this a good solution?

Information:
I use gcc 4.4.7 , distro is IPFire, arch is i586
If you need further information just ask.

Yours Sincerely jonaschl

**Comment by bmah888** [comment]
The part of the build that's falling over is the profiled iperf3 build.  If that's not important to you (and it probably shouldn't be to people who are not actively developing the iperf3 code) then you don't even need this part of the build to work right.  I think at that point in the build, the "normal" iperf3 executable should have been built already.

**Comment by jonaschl** [comment]
Seems to be fixed with gcc 4.9.3 or iperf 3.1.2. I do not know which change solve issues, but this is not so important.
